target/linux/bcm53xx/base-files/etc/board.d/02_network

```diff
@@ -31,6 +31,10 @@ bcm53xx_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0:lan" "1:lan" "2:lan" "3:lan" "4:wan" "5t@eth0"
 		;;
+	linksys,panamera)
+		ucidef_add_switch "switch1" \
+			"0:lan" "1:lan" "2:lan" "3:lan" "4:wan" "5t@eth0"
```

Comment:
Can you use full syntax, with LAN port number, please? That should be quite helpful on this complex device (with 2 switches). I mean `n:lan:m` as for some other devices.

What about other LAN ports? In DT I can see BCM53125 handling 5 LAN ports.

Reply:
I had this before, but I was told to get rid of it. I was in fact looking for an example how to use it for two switches but I could not find one. if  you can point me an example I can adapt it to my case.

`			"0:lan" "1:lan:7" "2:lan:4" "3:lan:8" "4:wan" "5t@eth0"`

As far as the other lan ports, I'm not sure the legacy switch driver can handle second switch except in dumb fwd mode, let me know if that's not the case. I've tested extensively with DSA and there is no easy way to configure DSA ports as OpenWrt lacks same framework as legacy switch driver.